fv3core/stencils/fv_dynamics.py

```diff
@@ -169,10 +169,19 @@ def do_dyn(state, comm):
         domain=grid.domain_shape_standard(),
     )
     print("DynCore", grid.rank)
+    if timer is not None:
+        # start dyn timer
+        timer.start("Dynamics")
     dyn_core.compute(state, comm)
+    if timer is not None:
+        # pause dyn timer
+        timer.stop("Dynamics")
     if not spec.namelist.inline_q and state.nq != 0:
         if spec.namelist.z_tracer:
             print("Tracer2D1L", grid.rank)
+            if timer is not None:
+                # start tracer timer
+                timer.start("Tracer advection")
```

Comment:
Can all these labels be changed to lowercase/pothole case, as we would if they were variable names?